pkg/docker/registry/registry.go

```diff
@@ -113,12 +122,28 @@ func getRegistryProxyEndpointFromKotsApplication(kotsApplication *kotsv1beta1.Ap
 	return proxyEndpoint, registryEndpoint
 }
 
-func getRegistryProxyInfoFromLicense(license *kotsv1beta1.License) *RegistryProxyInfo {
-	defaultInfo := &RegistryProxyInfo{
-		Upstream: "registry.replicated.com",
-		Registry: "registry.replicated.com",
-		Proxy:    "proxy.replicated.com",
+func getEmbeddedClusterRegistryProxyInfo() (*RegistryProxyInfo, error) {
+	info := &RegistryProxyInfo{
+		Upstream: util.DefaultReplicatedRegistryDomain(),
+	}
+
+	registryDomain := os.Getenv("REPLICATED_REGISTRY_DOMAIN")
+	if registryDomain == "" {
+		return nil, errors.New("REPLICATED_REGISTRY_DOMAIN environment variable is required")
 	}
+	info.Registry = registryDomain
```

Comment:
should also set `info.Upstream`

Reply:
i think the needs to remain registry.replicated.com